model/shared/src/main/scala/explore/model/ProgramDetails.scala

```diff
@@ -18,30 +17,30 @@ case class ProgramDetails(
   programType:    ProgramType,
   proposal:       Option[Proposal],
   proposalStatus: ProposalStatus,
-  pi:             ProgramUserWithRole,
+  pi:             Option[ProgramUserWithRole],
   users:          List[ProgramUserWithRole],
   invitations:    List[CoIInvitation],
   reference:      Option[ProgramReference]
 ) derives Eq:
-  val allUsers: NonEmptyList[ProgramUserWithRole] = NonEmptyList(pi, users)
+  val allUsers: List[ProgramUserWithRole] = pi.fold(users)(_ :: users)
```

Comment:
Isn't this concatenating users with users?

Reply:
pi with co-pi